src/DevBetterWeb.Vimeo/Services/VideoServices/CreateAnimatedThumbnailsService.cs

```diff
@@ -0,0 +1,84 @@
+﻿using System;
+using System.Threading;
+using System.Threading.Tasks;
+using DevBetterWeb.Vimeo.Models;
+using Microsoft.Extensions.Logging;
+
+namespace DevBetterWeb.Vimeo.Services.VideoServices;
+
+public class CreateAnimatedThumbnailsService
```

Comment:
Consider adding an interface if this is something we'll need to mock somewhere for test purposes, etc.

Reply:
Done